internal/bminventory/inventory.go

```diff
@@ -426,9 +427,9 @@ func (b *bareMetalInventory) formatIgnitionFile(cluster *common.Cluster, params
 		ignitionParams["ServiceIPs"] = dataurl.EncodeBytes([]byte(ignition.GetServiceIPHostnames(b.Config.ServiceIPs)))
 	}
 
-	if cluster.ImageInfo.StaticIpsConfig != "" {
+	if cluster.ImageInfo.StaticIpsConfig != "" && params.ImageCreateParams.ImageType != models.ImageTypeMinimalIso {
```

Comment:
just a nit, but maybe it'll be clearer to check '== models.ImageTypeFullIso', just to have the context of the relevant flow.